src/hooks/useInputValue.ts

```diff
@@ -1,26 +1,34 @@
 import { useState } from 'react';
 
-const useInputValue = ({ isValidateInput, isInputAvailableValue } = {}) => {
+interface Props {
+  isValidateInput: (value: string) => boolean;
+  isInputAvailableValue: (value: string) => boolean;
+}
+
+const useInputValue = ({
+  isValidateInput,
+  isInputAvailableValue,
+}: Partial<Props> = {}) => {
   const [value, setValue] = useState('');
   const [isError, setError] = useState(false);
 
-  const onChangeValue = ({ target }) => {
-    const value = target.value;
+  const onChangeValue = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
```

Comment:
사소한거긴 하지만, 구조 분해가 되는 변수는 분해 할당을 통해 만들어주시면 어떨까요!
```suggestion
    const { value } = e.target;
```